File: 4080-smallest-missing-multiple-of-k/smallest-missing-multiple-of-k.java
class Solution {
    public int missingMultiple(int[] nums, int k) {

        int idx = 1;

        while (true) {
            int target = k * idx;
            boolean found = false;

            for (int num : nums) {
                if (num == target) {
                    found = true;
                    break;
                }
            }

            if (!found) {
                return target;
            }

            idx++;
        }
    }
}

File: 4080-smallest-missing-multiple-of-k/smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        f=0
        idx=1
        v=0
        while f!=1:
            v=k*idx
            print(v)
            if v not in nums:
                f=1
                return(k*idx)
            idx+=1